reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/add-infra-node.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
Ya I was back and forth about creating a new stack or using pure ansible. By using a new stack we have a bit of consistency because we launch the everything that way but then you may have to call aws to increase your limits in the future.  I prefer the stack method because I have the assurance that the host and dns are ready by the time the next play starts without having to write a readiness check. Also, I can reuse some of the cloudformation role that is already established.